### PR TITLE
Added documentation for filter section

### PR DESCRIPTION
**Pull request by paulschuetz**

### CODE REVIEW

**Review by phyk — APPROVED** [review]
Good docs, maybe better formatting of newlines?

File: r_py_cheatsheet.md [hunk 1]
```diff
@@ -55,32 +55,39 @@ Working example (asap):
...
-Good help, tutorials:
+Purpose: Use filter find rows/cases where conditions are true.
+Official documentation: https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/filter
+Good help, tutorials: 
 
-1. ...
-2. ...
-3. ...
+1. [R for Data Science](https://r4ds.had.co.nz/transform.html#filter-rows-with-filter)
+2. https://www.guru99.com/r-select-filter-arrange.html#2
```
Comment (NicoLdrt) [inline r_py_cheatsheet.md:63]:
```suggestion
1. https://www.guru99.com/r-select-filter-arrange.html#2
```

File: r_py_cheatsheet.md [hunk 2]
```diff
@@ -55,32 +55,39 @@ Working example (asap):
 
 ### R
 
-Purpose:
-Official documentation:
-Good help, tutorials:
+Purpose: Use filter find rows/cases where conditions are true.
```
Comment (NicoLdrt) [inline r_py_cheatsheet.md]:
```suggestion
Purpose: Use "filter" to find rows/cases meeting certain conditions.<br>
```

File: r_py_cheatsheet.md [hunk 3]
```diff
@@ -55,32 +55,39 @@ Working example (asap):
...
+Official documentation: https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/filter
```
Comment (NicoLdrt) [inline r_py_cheatsheet.md]:
```suggestion
Official documentation: https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/filter<br>
```

File: r_py_cheatsheet.md [hunk 4]
```diff
@@ -55,32 +55,39 @@ Working example (asap):
...
+# filter for cars with city miles per gallon less than 20
+greenCars <- filter(mpg, cty < 20)
+print(greenCars)
 ```
 
 ### Python
 
-Purpose:
-Documentation:
+Purpose:  Use filter find rows/cases where certain conditions are meet.
```
Comment (NicoLdrt) [inline r_py_cheatsheet.md]:
```suggestion
Purpose:  Use "filter" to find rows/cases meeting certain conditions.<br>
```

File: r_py_cheatsheet.md [hunk 5]
```diff
@@ -55,32 +55,39 @@ Working example (asap):
...
+Documentation: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.filter.html
```
Comment (NicoLdrt) [inline r_py_cheatsheet.md]:
```suggestion
Documentation: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.filter.html<br>
```

**Review by NicoLdrt — CHANGES_REQUESTED** [review]

File: r_py_cheatsheet.md (hunk 1 above)
Comment (paulschuetz) [reply r_py_cheatsheet.md:63]: Do not unterstand this suggestion

File: r_py_cheatsheet.md (hunk 4 above)
Comment (paulschuetz) [reply r_py_cheatsheet.md]: text is nice but you do not need that "\<br\>" I think.

**Comment by paulschuetz** [comment]
@NicoLdrt  @phyk  thanks for the review. Added your changes!

**Review by JBraun253 — APPROVED** [review]
Looks great!

**Review by phyk — APPROVED** [review]

**Review by phyk — APPROVED** [review]

**Review by NicoLdrt — APPROVED** [review]

**Pull request merged**